packages/members-api/lib/repositories/member.js

```diff
@@ -712,10 +724,9 @@ module.exports = class MemberRepository {
                 id: model.id
             });
 
-            if (model.get('plan_id') !== updated.get('plan_id') || model.get('status') !== updated.get('status') || model.get('cancel_at_period_end') !== updated.get('cancel_at_period_end')) {
-                // TODO: we should use the previously saved MRR in the future as soon as the MRR column is 'stable' and migrated correctly
-                const originalMrrDelta = this.getMRR({interval: model.get('plan_interval'), amount: model.get('plan_amount'), status: model.get('status'), canceled: model.get('cancel_at_period_end')});
-                const updatedMrrDelta = this.getMRR({interval: updated.get('plan_interval'), amount: updated.get('plan_amount'), status: updated.get('status'), canceled: updated.get('cancel_at_period_end')});
+            if (model.get('mrr') !== updated.get('mrr') || model.get('plan_id') !== updated.get('plan_id') || model.get('status') !== updated.get('status') || model.get('cancel_at_period_end') !== updated.get('cancel_at_period_end')) {
```

Comment:
It's worth considering simplifying this conditional to check just mrr and status, but it's not worth doing it now!

Reply:
Yes, you are right! I don't mind to do it now 😊

Reply:
We still need to include `cancel_at_period_end` until the dashboardv5 flag is dropped